docpages/example_programs/the_basics/editing-objects.md

```diff
@@ -0,0 +1,87 @@
+\page editing-objects Editing Channel and Message
+
+Sometimes we need to update an object, such as message or channel. At first, it might seem confusing, but it's actually really simple! It's done with \ref callback-functions callbacks. You just need to use an object with identical properties you don't need to update.
```

Comment:
Just realised I've messed up the ref for ya, needs to be only be `\ref callback-functions`

Reply:
But if so, this means that this will appear as [Using Callback Functions](Link), right? In that case the text is "It's done with Using Callback Functions" which doesn't really make sense. Should I use `[]()` instead?

Reply:
> But if so, this means that this will appear as `[Using Callback Functions](Link)`, right? In that case the text is "It's done with Using Callback Functions" which doesn't really make sense. Should I use `[]()` instead?

Yes, it will appear with "Using Callback Functions". No you shouldn't use `[]()`, that doesn't follow doxygen. You could remove the `It's done with callbacks` part and move that into a note below using `\note` and then write something like `\note This page uses callbacks. If you need more information, visit \ref callback-functions`

Reply:
> Yes, it will appear with "Using Callback Functions". No you shoul...

Sounds good